src/mod/endpoints/mod_loopback/mod_loopback.c

```diff
@@ -1170,6 +1170,9 @@ static switch_call_cause_t channel_outgoing_channel(switch_core_session_t *sessi
 				switch_channel_set_variable(channel, "loopback_leg", "A");
 				switch_channel_set_variable(channel, "is_loopback", "1");
 			}
+
+			switch_channel_set_variable(channel, "loopback_initial_codec", switch_event_get_header(var_event, "loopback_initial_codec"));
```

Comment:
I think this should old be set if `loopback_initial_codec` is passed into `var_event`.

Reply:
ok, i just change to check if its set.